flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -119,6 +120,7 @@ def serialize_all(pkgs: List[str], folder: str = None, mode: SerializationMode =
                 o.resource_type, _PROJECT_PLACEHOLDER, _DOMAIN_PLACEHOLDER, name, _VERSION_PLACEHOLDER
             )
             loaded_entities.append(o)
+            ctx.registration_settings.add_instance_var(InstanceVar(module=m, name=k, o=o))
```

Comment:
interesting, we only add these instance vars at serialization?

Reply:
yes